src/main/java/com/julienvey/trello/Trello.java

```diff
@@ -144,6 +144,8 @@
 
     /* Organizations */
 
+    Organization getOrganization(String organizationId, Argument... args);
```

Comment:
what happens when the requested org does not exist? 
the method should return Optional<..>

Reply:
An exception will be triggered in this case. (exceptions may differ)
For example : `com.julienvey.trello.NotFoundException: Resource not found:`

I'm not sure about the relevance of the `Optional` return.
`Optional` is not in accordance with other methods such as `getAction`, `getBoard`, etc, which do not use this concept.

Reply:
ah, indeed. this project uses exceptions instead of Optional<>.
let's document this behavior in the javadoc for this method